### ISSUE TITLE
ticker.info fails for stock RBREW.CO

### ISSUE THREAD

**Issue by KennethNielsen** [ranaroussi/yfinance#349]
When I try to retrieve info for the RBREW.CO stock I get the following traceback:

```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kenneth/.local/lib/python3.6/site-packages/yfinance/ticker.py", line 138, in info
    return self.get_info()
  File "/home/kenneth/.local/lib/python3.6/site-packages/yfinance/base.py", line 415, in get_info
    self._get_fundamentals(proxy)
  File "/home/kenneth/.local/lib/python3.6/site-packages/yfinance/base.py", line 286, in _get_fundamentals
    self._institutional_holders = holders[1]
IndexError: list index out of range
```

The package works fine for other stock symbols. If you need any more information let me know.

**Comment by rodrigobercini** [comment]
Hey there, try applying the fixes described in PR #179.

Just be aware that that fix returns inconsistent data for non-US tickers.

**Comment by KennethNielsen** [comment]
@rodrigobercini well, this is for a stock on the danish stock exchange, so obviously it is a concern with inconsistent data. In any case the fix works and at least the bid price seems correct.
